drivers/espi/espi_mchp_xec_v2.c

```diff
@@ -1561,3 +1561,282 @@ static int espi_xec_init(const struct device *dev)
 
 	return ret;
 }
+
```

Comment:
Consider adding all implementation in separate file

Reply:
Actually this implementation could be better in SPI flash driver that is a client from eSPI driver  to ease maintenance of the eSPI driver 
@scottwcpg  you have been trying to decouple other functionality, this goes in the opposite direction

Reply:
i had originally considered this as well, but the way it ended up was just easier to get going and get some validation of the POC.  I think that makes more sense, i'll see if i can do it that way without too much trouble.